cypress/e2e/tests/migration/applicationinventory/analysis/auto_tagging_filter.test.ts

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import { login, getRandomApplicationData, getRandomAnalysisData } from "../../../../../utils/utils";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import { CredentialType, UserCredentials } from "../../../../types/constants";
+import * as data from "../../../../../utils/data_utils";
+import { CredentialsSourceControlUsername } from "../../../../models/administration/credentials/credentialsSourceControlUsername";
+import { Tag } from "../../../../models/migration/controls/tags";
+import { TagCategory } from "../../../../models/migration/controls/tagcategory";
+import { appDetailsView } from "../../../../views/applicationinventory.view";
+import { SEC } from "../../../../types/constants";
+let source_credential;
+let tagCategory;
+let tag;
+
+describe(["@tier3"], "Source Analysis", () => {
```

Comment:
"Source Analysis" doesn't sound right .

You can name it "Auto tagging applications and Filtering based on tags"